locales/el/el.json

```diff
@@ -1,618 +1,618 @@
-    "Create": "Create",
-    "Create & Add Another": "Create & Add Another",
-    "Create :resource": "Create :resource",
-    "Create a new team to collaborate with others on projects.": "Create a new team to collaborate with others on projects.",
-    "Create Account": "Create Account",
-    "Create API Token": "Create API Token",
-    "Create New Team": "Create New Team",
-    "Create Team": "Create Team",
-    "Created.": "Created.",
-    "Croatia": "Croatia",
-    "Cuba": "Cuba",
-    "Curaçao": "Curaçao",
-    "Current Password": "Current Password",
-    "Customize": "Customize",
-    "Cyprus": "Cyprus",
-    "Czech Republic": "Czechia",
-    "Dashboard": "Dashboard",
-    "December": "December",
-    "Decrease": "Decrease",
-    "Delete": "Delete",
-    "Delete Account": "Delete Account",
-    "Delete API Token": "Delete API Token",
-    "Delete File": "Delete File",
-    "Delete Resource": "Delete Resource",
-    "Delete Selected": "Delete Selected",
-    "Delete Team": "Delete Team",
-    "Denmark": "Denmark",
-    "Detach": "Detach",
-    "Detach Resource": "Detach Resource",
-    "Detach Selected": "Detach Selected",
-    "Details": "Details",
-    "Disable": "Disable",
-    "Djibouti": "Djibouti",
-    "Do you really want to leave? You have unsaved changes.": "Do you really want to leave? You have unsaved changes.",
-    "Dominica": "Dominica",
-    "Dominican Republic": "Dominican Republic",
-    "Done.": "Done.",
-    "Download": "Download",
+    "could not be found.": "δεν βρέθηκε.",
+    "Create": "Δημιουργήσετε",
+    "Create & Add Another": "Δημιουργία & Προσθήκη Άλλου",
+    "Create :resource": "Δημιουργία :resource",
+    "Create a new team to collaborate with others on projects.": "Δημιουργήστε μια νέα ομάδα για να συνεργαστείτε με άλλους σε έργα.",
+    "Create Account": "Δημιουργία Λογαριασμού",
+    "Create API Token": "Δημιουργία Token API",
+    "Create New Team": "Δημιουργία Νέας Ομάδας",
+    "Create Team": "Δημιουργία Ομάδας",
+    "Created.": "Δημιουργήθηκε.",
+    "Croatia": "Κροατία",
+    "Cuba": "Κούβα",
+    "Curaçao": "Κουρασάο",
+    "Current Password": "Τρέχων Κωδικός Πρόσβασης",
+    "Customize": "Προσαρμογή",
+    "Cyprus": "Κύπρος",
+    "Czech Republic": "Τσεχία",
+    "Dashboard": "Πίνακας",
+    "December": "Δεκεμβρίου",
+    "Decrease": "Μειώσετε",
+    "Delete": "Διαγράψετε",
+    "Delete Account": "Διαγραφή Λογαριασμού",
+    "Delete API Token": "Διαγραφή Token API",
+    "Delete File": "Διαγραφή Αρχείου",
+    "Delete Resource": "Διαγραφή Πόρου",
+    "Delete Selected": "Διαγραφή Επιλεγμένων",
+    "Delete Team": "Διαγραφή Ομάδας",
+    "Denmark": "Δανία",
+    "Detach": "Αποσυνδέσετε",
+    "Detach Resource": "Αποσύνδεση Πόρου",
+    "Detach Selected": "Αποσύνδεση Επιλεγμένων",
+    "Details": "Στοιχεία",
+    "Disable": "Απενεργοποιήσετε",
+    "Djibouti": "Τζιμπουτί",
+    "Do you really want to leave? You have unsaved changes.": "Θέλεις πραγματικά να φύγεις; Έχετε μη αποθηκευμένες αλλαγές.",
```

Comment:
First sentence addresses the user informally with the question, but then switches to a formal address stating the warning. If it makes sense. Both sentences should address the user using the same way.
Either: "Θέλεις πραγματικά να φύγεις; Έχεις μη αποθηκευμένες αλλαγές."
or: "Θέλετε πραγματικά να φύγετε; Έχετε μη αποθηκευμένες αλλαγές."